Sources/Swift/Tools/Batcher/InMemoryBatchBuffer.swift

```diff
@@ -0,0 +1,27 @@
+struct InMemoryBatchBuffer<Item: Encodable>: BatchBuffer {
+    private var elements: [Data] = []
+    var itemsDataSize: Int = 0
+
+    init() {}
+
+    mutating func append(_ item: Item) throws {
+        let encoder = JSONEncoder()
+        encoder.dateEncodingStrategy = .secondsSince1970
```

Comment:
`l`: Wouldn't it make sense to init the `JSONEncoder` once and keep it as a private property?